rtengine/rawimagesource.cc

```diff
@@ -5307,8 +5307,8 @@ void RawImageSource::ItcWB(bool extra, double &tempref, double &greenref, double
         Glib::ustring profuse;
         profuse = "JDCmax";
 
-        int limx = 0.05f;
-        int limy = 0.04f;
+        float limx = 0.05f;
```

Comment:
It seems like the one of the algorithms introduced in #6643 didn't work well because of this convertion to 0 all the time

Reply:
@Desmis Does this behave well after the fix?

Reply:
solved ([comment](https://github.com/Beep6581/RawTherapee/pull/6873#issuecomment-1784014146))